{
  "Faults": [
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-07T08:00:00.000+0000",
      "latitude": 50.77641522165823,
      "estimatedRestoration": "2022-02-11T17:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205646'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205646",
      "reference": "205646",
      "name": "BH21 & DT11 Areas",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH21 3QZ",
        "DT11 9FQ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -2.0507082705621147,
          50.77641522165823
        ],
        "type": "Point"
      },
      "UUID": "cf804b70-e38f-469c-8e33-cb19f9c5636a",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.0507082705621147,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-07T08:30:00.000+0000",
      "latitude": 50.93421335057585,
      "estimatedRestoration": "2022-02-11T14:30:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205299'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205299",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 50.93354,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.82025
            },
            {
              "py/object": "Point",
              "latitude": 50.93661,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.81364
            },
            {
              "py/object": "Point",
              "latitude": 50.93952,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.80003
            },
            {
              "py/object": "Point",
              "latitude": 50.93077,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.79851
            },
            {
              "py/object": "Point",
              "latitude": 50.93158,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.81569
            },
            {
              "py/object": "Point",
              "latitude": 50.93354,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.82025
            }
          ],
          "subCodes": [
            "SP6 1DT",
            "SP6 1JR",
            "SP6 1NH",
            "SP6 1NS",
            "SP6 1NT",
            "SP6 1NU",
            "SP6 1NX",
            "SP6 1NY",
            "SP6 1PA"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "205299",
      "name": "SP6 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SP6 1DT",
        "SP6 1JR",
        "SP6 1NH",
        "SP6 1NS",
        "SP6 1NT",
        "SP6 1NU",
        "SP6 1NX",
        "SP6 1NY",
        "SP6 1PA"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -1.82025,
              50.93354
            ],
            [
              -1.81364,
              50.93661
            ],
            [
              -1.80003,
              50.93952
            ],
            [
              -1.79851,
              50.93077
            ],
            [
              -1.81569,
              50.93158
            ],
            [
              -1.82025,
              50.93354
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "e9a87bee-4f6e-4bd7-9a93-223e6f842205",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.8087316358276926,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-09T07:58:00.000+0000",
      "latitude": 50.89565629518321,
      "estimatedRestoration": "2022-02-09T11:00:00.000+0000",
      "message": "Weâ€™re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JM0627'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JM0627",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 50.92276,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.46904
            },
            {
              "py/object": "Point",
              "latitude": 50.89565629518321,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.3548913428445646
            },
            {
              "py/object": "Point",
              "latitude": 50.89567,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.34565
            },
            {
              "py/object": "Point",
              "latitude": 50.8953,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.3449
            },
            {
              "py/object": "Point",
              "latitude": 50.89178,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.33844
            },
            {
              "py/object": "Point",
              "latitude": 50.89234,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.34938
            },
            {
              "py/object": "Point",
              "latitude": 50.89243,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.34995
            },
            {
              "py/object": "Point",
              "latitude": 50.92276,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.46904
            }
          ],
          "subCodes": [
            "SO16 9AT",
            "SO19 0NQ",
            "SO19 0NS",
            "SO19 0QZ",
            "SO19 0SX",
            "SO19 0TQ",
            "SO19 0TR",
            "SO19 9AT",
            "SO19 9BJ",
            "SO19 9LT",
            "SO19 9LX",
            "SO19 9LY",
            "SO19 9LZ",
            "SO19 9PW",
            "SO19 9QY",
            "SO19 9QZ",
            "SO19 9UW"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "JM0627",
      "name": "SO16 & SO19 Areas",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SO16 9AT",
        "SO19 0NQ",
        "SO19 0NS",
        "SO19 0QZ",
        "SO19 0SX",
        "SO19 0TQ",
        "SO19 0TR",
        "SO19 9AT",
        "SO19 9BJ",
        "SO19 9LT",
        "SO19 9LX",
        "SO19 9LY",
        "SO19 9LZ",
        "SO19 9PW",
        "SO19 9QY",
        "SO19 9QZ",
        "SO19 9UW"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -1.46904,
              50.92276
            ],
            [
              -1.3548913428445646,
              50.89565629518321
            ],
            [
              -1.34565,
              50.89567
            ],
            [
              -1.3449,
              50.8953
            ],
            [
              -1.33844,
              50.89178
            ],
            [
              -1.34938,
              50.89234
            ],
            [
              -1.34995,
              50.89243
            ],
            [
              -1.46904,
              50.92276
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-02-09T09:20:00.000+0000",
      "UUID": "bb282ec1-036c-4e4c-ac8d-c51496971851",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "evChargersAffected": [
        "939bae64-41e3-43c9-8860-0715397f773b",
        "550ea32b-20ed-4699-bdf4-875a6782402e",
        "0c53b1c1-dc14-45b3-8d8a-f9ab6d1a37ce",
        "c0d4dd22-15b2-480d-ae50-5e3f17dc31a8",
        "86b1a320-fd33-400f-94ac-845d1f8b852a",
        "010220aa-de17-489a-b1a4-61c0dc01b72a"
      ],
      "longitude": -1.3548913428445646,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-09T08:30:00.000+0000",
      "latitude": 51.03119182896624,
      "estimatedRestoration": "2022-02-09T12:30:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205247'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205247",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.03141,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.15182
            },
            {
              "py/object": "Point",
              "latitude": 51.0318,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.15169
            },
            {
              "py/object": "Point",
              "latitude": 51.03224,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.14904
            },
            {
              "py/object": "Point",
              "latitude": 51.03014,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.14735
            },
            {
              "py/object": "Point",
              "latitude": 51.02988,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.15059
            },
            {
              "py/object": "Point",
              "latitude": 51.03141,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.15182
            }
          ],
          "subCodes": [
            "SP7 9AZ",
            "SP7 9BA",
            "SP7 9BB",
            "SP7 9BE",
            "SP7 9BG"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "205247",
      "name": "SP7 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SP7 9AZ",
        "SP7 9BA",
        "SP7 9BB",
        "SP7 9BE",
        "SP7 9BG"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -2.15182,
              51.03141
            ],
            [
              -2.15169,
              51.0318
            ],
            [
              -2.14904,
              51.03224
            ],
            [
              -2.14735,
              51.03014
            ],
            [
              -2.15059,
              51.02988
            ],
            [
              -2.15182,
              51.03141
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "7d2f0716-387a-466d-8360-4e4cd2f7d7e4",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.150224938510839,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-09T06:38:00.000+0000",
      "latitude": 51.205322134442035,
      "estimatedRestoration": "2022-02-09T11:00:00.000+0000",
      "message": "Weâ€™re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JM0612'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JM0612",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.20585,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.80436
            },
            {
              "py/object": "Point",
              "latitude": 51.20489,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.80346
            },
            {
              "py/object": "Point",
              "latitude": 51.20534,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.80133
            },
            {
              "py/object": "Point",
              "latitude": 51.20585,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.80436
            }
          ],
          "subCodes": [
            "GU9 8RN",
            "GU9 8RP",
            "GU9 8RR",
            "GU9 8RW"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "JM0612",
      "name": "GU9 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "GU9 8RN",
        "GU9 8RP",
        "GU9 8RR",
        "GU9 8RW"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -0.80436,
              51.20585
            ],
            [
              -0.80346,
              51.20489
            ],
            [
              -0.80133,
              51.20534
            ],
            [
              -0.80436,
              51.20585
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-02-09T08:15:00.000+0000",
      "UUID": "09aebf36-319d-4043-a4f7-9ac11f1489ae",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8029345356146204,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-09T07:58:00.000+0000",
      "latitude": 50.859456319730455,
      "estimatedRestoration": "2022-02-09T11:30:00.000+0000",
      "message": "Weâ€™re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JM0632'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JM0632",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 50.86191,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.32761
            },
            {
              "py/object": "Point",
              "latitude": 50.86104,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.31899
            },
            {
              "py/object": "Point",
              "latitude": 50.85879,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.31758
            },
            {
              "py/object": "Point",
              "latitude": 50.85582,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.31689
            },
            {
              "py/object": "Point",
              "latitude": 50.86077,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.32669
            },
            {
              "py/object": "Point",
              "latitude": 50.86191,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.32761
            }
          ],
          "subCodes": [
            "SO31 4JD",
            "SO31 4JE",
            "SO31 4JF",
            "SO31 4JR",
            "SO31 4JS",
            "SO31 4QA",
            "SO31 4RT",
            "SO31 4RX"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "JM0632",
      "name": "SO31 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SO31 4JD",
        "SO31 4JE",
        "SO31 4JF",
        "SO31 4JR",
        "SO31 4JS",
        "SO31 4QA",
        "SO31 4RT",
        "SO31 4RX"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -1.32761,
              50.86191
            ],
            [
              -1.31899,
              50.86104
            ],
            [
              -1.31758,
              50.85879
            ],
            [
              -1.31689,
              50.85582
            ],
            [
              -1.32669,
              50.86077
            ],
            [
              -1.32761,
              50.86191
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-02-09T09:30:00.000+0000",
      "UUID": "5186e42e-273d-4966-a356-403d6a6318e7",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3207230298029675,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-09T09:00:00.000+0000",
      "latitude": 50.727827725665506,
      "estimatedRestoration": "2022-02-09T12:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205004'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205004",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 50.72852,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.78572
            },
            {
              "py/object": "Point",
              "latitude": 50.72618,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.78486
            },
            {
              "py/object": "Point",
              "latitude": 50.72824,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.78045
            },
            {
              "py/object": "Point",
              "latitude": 50.72841,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.78167
            },
            {
              "py/object": "Point",
              "latitude": 50.72852,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.78572
            }
          ],
          "subCodes": [
            "BH6 4HS",
            "BH6 4JX",
            "BH6 4JY",
            "BH6 4JZ",
            "BH6 4LH",
            "BH6 4LQ"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "205004",
      "name": "BH6 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BH6 4HS",
        "BH6 4JX",
        "BH6 4JY",
        "BH6 4JZ",
        "BH6 4LH",
        "BH6 4LQ"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -1.78572,
              50.72852
            ],
            [
              -1.78486,
              50.72618
            ],
            [
              -1.78045,
              50.72824
            ],
            [
              -1.78167,
              50.72841
            ],
            [
              -1.78572,
              50.72852
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "d40fdced-af62-487b-a033-f85d2a82e6e8",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.7838144998811654,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-09T09:00:00.000+0000",
      "latitude": 51.410631001044685,
      "estimatedRestoration": "2022-02-09T15:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205403'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205403",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.41076,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.84439
            },
            {
              "py/object": "Point",
              "latitude": 51.41178,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.8403
            },
            {
              "py/object": "Point",
              "latitude": 51.41188,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.83981
            },
            {
              "py/object": "Point",
              "latitude": 51.41174,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.83787
            },
            {
              "py/object": "Point",
              "latitude": 51.4117,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.83728
            },
            {
              "py/object": "Point",
              "latitude": 51.40969,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.83983
            },
            {
              "py/object": "Point",
              "latitude": 51.40957,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.84271
            },
            {
              "py/object": "Point",
              "latitude": 51.41055,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.84421
            },
            {
              "py/object": "Point",
              "latitude": 51.41076,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.84439
            }
          ],
          "subCodes": [
            "RG40 1BJ",
            "RG40 1BS",
            "RG40 2AD",
            "RG40 2AE",
            "RG40 2AG",
            "RG40 2AJ",
            "RG40 2AL",
            "RG40 2AX",
            "RG41 2WA",
            "RG41 2XP",
            "RG41 2XR",
            "RG41 2XT",
            "RG41 2XU",
            "RG41 2XX",
            "RG41 2YD"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "205403",
      "name": "RG40 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG40 1BJ",
        "RG40 1BS",
        "RG40 2AD",
        "RG40 2AE",
        "RG40 2AG",
        "RG40 2AJ",
        "RG40 2AL",
        "RG40 2AX",
        "RG41 2WA",
        "RG41 2XP",
        "RG41 2XR",
        "RG41 2XT",
        "RG41 2XU",
        "RG41 2XX",
        "RG41 2YD"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -0.84439,
              51.41076
            ],
            [
              -0.8403,
              51.41178
            ],
            [
              -0.83981,
              51.41188
            ],
            [
              -0.83787,
              51.41174
            ],
            [
              -0.83728,
              51.4117
            ],
            [
              -0.83983,
              51.40969
            ],
            [
              -0.84271,
              51.40957
            ],
            [
              -0.84421,
              51.41055
            ],
            [
              -0.84439,
              51.41076
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "b41adf3e-c804-402a-9d8b-2a9cc39248b0",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8413359098282177,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-09T09:00:00.000+0000",
      "latitude": 51.339407575996226,
      "estimatedRestoration": "2022-02-09T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '204735'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/204735",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.33748,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.31417
            },
            {
              "py/object": "Point",
              "latitude": 51.34149,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.30761
            },
            {
              "py/object": "Point",
              "latitude": 51.3419,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.30683
            },
            {
              "py/object": "Point",
              "latitude": 51.3403,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.30435
            },
            {
              "py/object": "Point",
              "latitude": 51.3395,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.30372
            },
            {
              "py/object": "Point",
              "latitude": 51.33515,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.30798
            },
            {
              "py/object": "Point",
              "latitude": 51.33553,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.3113
            },
            {
              "py/object": "Point",
              "latitude": 51.33748,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -2.31417
            }
          ],
          "subCodes": [
            "BA2 7PL",
            "BA2 7TT",
            "BA2 7TU",
            "BA2 7TX",
            "BA2 7TY",
            "BA2 7TZ",
            "BA2 7UA",
            "BA2 7UB",
            "BA2 7UP",
            "BA2 7UR",
            "BA2 7WA",
            "BA2 7WB",
            "BA2 7WD",
            "BA2 7WE",
            "BA2 7WF",
            "BA2 7WR"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "204735",
      "name": "BA2 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "BA2 7PL",
        "BA2 7TT",
        "BA2 7TU",
        "BA2 7TX",
        "BA2 7TY",
        "BA2 7TZ",
        "BA2 7UA",
        "BA2 7UB",
        "BA2 7UP",
        "BA2 7UR",
        "BA2 7WA",
        "BA2 7WB",
        "BA2 7WD",
        "BA2 7WE",
        "BA2 7WF",
        "BA2 7WR"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -2.31417,
              51.33748
            ],
            [
              -2.30761,
              51.34149
            ],
            [
              -2.30683,
              51.3419
            ],
            [
              -2.30435,
              51.3403
            ],
            [
              -2.30372,
              51.3395
            ],
            [
              -2.30798,
              51.33515
            ],
            [
              -2.3113,
              51.33553
            ],
            [
              -2.31417,
              51.33748
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "740d9221-4bde-451b-8373-2d887be3b476",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.307466006814713,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-09T09:00:00.000+0000",
      "latitude": 51.19111562504118,
      "estimatedRestoration": "2022-02-09T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205155'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205155",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.18847,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.66661
            },
            {
              "py/object": "Point",
              "latitude": 51.19738,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.6632
            },
            {
              "py/object": "Point",
              "latitude": 51.1947,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.65459
            },
            {
              "py/object": "Point",
              "latitude": 51.18744,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.66289
            },
            {
              "py/object": "Point",
              "latitude": 51.18847,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.66661
            }
          ],
          "subCodes": [
            "GU8 6AE",
            "GU8 6AN",
            "GU8 6AY",
            "GU8 6BA",
            "GU8 6BD",
            "GU8 6BE",
            "GU8 6BG",
            "GU8 6BQ",
            "GU8 6BY"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "205155",
      "name": "GU8 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "GU8 6AE",
        "GU8 6AN",
        "GU8 6AY",
        "GU8 6BA",
        "GU8 6BD",
        "GU8 6BE",
        "GU8 6BG",
        "GU8 6BQ",
        "GU8 6BY"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -0.66661,
              51.18847
            ],
            [
              -0.6632,
              51.19738
            ],
            [
              -0.65459,
              51.1947
            ],
            [
              -0.66289,
              51.18744
            ],
            [
              -0.66661,
              51.18847
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "dad0d1b5-6f6f-42e8-9f7e-cb89ca5ed0ee",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.6619589474971281,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-09T09:00:00.000+0000",
      "latitude": 51.374330506539614,
      "estimatedRestoration": "2022-02-09T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '204881'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/204881",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 51.37517,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -0.89685
            }
          ],
          "name": "RG40 4QX",
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "204881",
      "name": "RG40 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "RG40 4QX"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.8944200242183524,
          51.374330506539614
        ],
        "type": "Point"
      },
      "UUID": "87ef9dde-8c81-4a0b-b565-a46ad8adff7c",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8944200242183524,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-09T09:00:00.000+0000",
      "latitude": 50.77126343825906,
      "estimatedRestoration": "2022-02-09T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '204651'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/204651",
      "Areas": [
        {
          "py/object": "Area",
          "Points": [
            {
              "py/object": "Point",
              "latitude": 50.77216,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.60483
            },
            {
              "py/object": "Point",
              "latitude": 50.77126343825906,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.6029461053851657
            },
            {
              "py/object": "Point",
              "latitude": 50.77,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.60099
            },
            {
              "py/object": "Point",
              "latitude": 50.77216,
              "py/class": "http://opengrid.com/givr/1.0#//Point",
              "longitude": -1.60483
            }
          ],
          "subCodes": [
            "SO41 6DD",
            "SO41 6DN"
          ],
          "py/class": "http://opengrid.com/givr/1.0#//Area"
        }
      ],
      "reference": "204651",
      "name": "SO41 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SO41 6DD",
        "SO41 6DN"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          [
            [
              -1.60483,
              50.77216
            ],
            [
              -1.6029461053851657,
              50.77126343825906
            ],
            [
              -1.60099,
              50.77
            ],
            [
              -1.60483,
              50.77216
            ]
          ]
        ],
        "type": "Polygon"
      },
      "UUID": "f3dac6b5-ae4e-4bfa-a44e-5cd72d0c52fc",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.6029461053851657,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-08T21:26:00.000+0000",
      "latitude": 55.66522249220647,
      "estimatedRestoration": "2022-02-09T12:00:00.000+0000",
      "message": "Weâ€™re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JM0604'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JM0604",
      "reference": "JM0604",
      "name": "PA42 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "PA42 7AQ",
        "PA42 7AS",
        "PA42 7AW",
        "PA42 7DD",
        "PA42 7DF"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -6.223598161452186,
          55.66522249220647
        ],
        "type": "Point"
      },
      "UUID": "70ecfa8e-ffc2-4f8b-87f4-617ddc6978a4",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -6.223598161452186,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-09T08:52:00.000+0000",
      "latitude": 58.14190743141903,
      "estimatedRestoration": "2022-02-09T12:00:00.000+0000",
      "message": "Weâ€™re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JM0653'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JM0653",
      "reference": "JM0653",
      "name": "HS2 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "HS2 9NH",
        "HS2 9NJ",
        "HS2 9NL",
        "HS2 9NN",
        "HS2 9NP",
        "HS2 9NW"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -6.425099184735321,
          58.14190743141903
        ],
        "type": "Point"
      },
      "UUID": "e285b93d-d4cf-4a14-a117-7166181d9b5a",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -6.425099184735321,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-09T09:00:00.000+0000",
      "latitude": 57.428643199717385,
      "estimatedRestoration": "2022-02-09T15:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205116'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205116",
      "reference": "205116",
      "name": "IV2 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "IV2 6DL"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -4.2824887256531685,
          57.428643199717385
        ],
        "type": "Point"
      },
      "UUID": "0433653e-266b-4093-8b54-a6cfa6211979",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -4.2824887256531685,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-09T09:00:00.000+0000",
      "latitude": 56.09853501365599,
      "estimatedRestoration": "2022-02-09T16:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205267'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205267",
      "reference": "205267",
      "name": "G83 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "G83 8NN",
        "G83 8NT",
        "G83 8NU",
        "G83 8NX",
        "G83 8NY",
        "G83 8NZ",
        "G83 8PA",
        "G83 8PB",
        "G83 8PD",
        "G83 8PG",
        "G83 8RH",
        "G83 8TU"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -4.646332034159316,
          56.09853501365599
        ],
        "type": "Point"
      },
      "UUID": "9514fb45-b544-4210-9ced-3dd439158101",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -4.646332034159316,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-02-09T09:00:00.000+0000",
      "latitude": 55.99924088695768,
      "estimatedRestoration": "2022-02-09T15:00:00.000+0000",
      "message": "We're sorry for the loss of supply. The reason you're currently without power is a planned outage in the areas listed. We have turned off your electricity supply to allow us to safely carry out essential work to our electrical equipment. By doing this, it'll help us to improve the reliability of our service to you for the future. If you have any questions or require more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) with reference  '205139'. You will have received a letter from us confirming your power will be interrupted with specific information relating to your property. If you did not receive this letter, you can assume your power will not be interrupted.",
      "type": "PSI",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/205139",
      "reference": "205139",
      "name": "PA31 Area",
      "networkId": "com.sse.ssepd.shepd",
      "affectedAreas": [
        "PA30 8ER",
        "PA31 8NH",
        "PA31 8NY"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -5.455577953565955,
          55.99924088695768
        ],
        "type": "Point"
      },
      "UUID": "3a6ca580-f86a-4304-98ea-a97694c37313",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -5.455577953565955,
      "resolved": false
    }
  ]
}
